exposure-notification/src/main/java/fi/thl/covid19/exposurenotification/efgs/Callback.java

```diff
@@ -0,0 +1,12 @@
+package fi.thl.covid19.exposurenotification.efgs;
+
+public class Callback {
+
+    public final String callbackId;
+    public final String url;
+
+    public Callback(String callbackId, String url) {
+        this.callbackId = callbackId;
```

Comment:
requireNonNull

Reply:
I added this. I'm starting to think that it's better having traditional nulls than this kind of boilerplate....